### PR TITLE
Big changes in the code base

### PR DESCRIPTION
**Pull request by jDan735**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,52 @@
...
+| Language      | Status   | Language      | Status    |
+|---------------|----------|---------------|-----------|
+| 🇷🇺 Russian    | 👍 stable | 🇩🇪 German     | ⚠️ beta   |
 | 🇺🇦 Ukrainian  | 👍 stable | 🇬🇧 English    | 👍 stable |
-| 🇫🇷 French     | ❌ none   | 🇪🇸 Spanish    | ❌ none   |
-| 🇨🇳 Chinese    | ❌ none   | 🇵🇹 Portuguese | ❌ none   |
-| 🇵🇱 Polish     | ❌ none   | 🇮🇹 Italien    | ❌ none   |
+| 🇫🇷 French     | ⚠️ beta   | 🇪🇸 Spanish    | 🗓 plans  |
+| 🇨🇳 Chinese    | ❌ none   | 🇵🇹 Portuguese | 🗓 plans  |
+| 🇵🇱 Polish     | 🗓 plans  | 🇮🇹 Italien    | ⚠️ beta   |
```
Comment (0x0bloodyknight) [inline README.md]: Italien or Italian?🤔

File: README.md [hunk 2]
```diff
@@ -1,37 +1,52 @@
...
-python ropasci.py
-```
-### ⚠️ Start with debug
-```
-python ropasci.py dev
+python kanobu.py
 ```
+### ⚙️ Options
+#### dev
+For get developers functions (`log`, `clog`), and logs about variables and other parameters
```
Comment (0x0bloodyknight) [inline README.md]: "To get" or "For getting"

File: locale/ua.cson [hunk 3]
```diff
@@ -1,4 +1,6 @@
-name: "Український"
+lang:
+	name: "Український"
```
Comment (0x0bloodyknight) [inline locale/ua.cson]: Українська мова🌚

**Review by 0x0bloodyknight — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jDan735) [reply README.md]: Italian

File: README.md (hunk 2 above)
Comment (jDan735) [reply README.md]: To get

File: locale/ua.cson (hunk 3 above)
Comment (jDan735) [reply locale/ua.cson]: Ну да 🤡

**Pull request merged**